.github/checkgroup.yml

```diff
@@ -1,20 +1,18 @@
 custom_service_name: "Lightning CI required checker"
+# For security reasons, configuration is only loaded from the repository's default branch,
```

Comment:
can you rebase with master? maybe you picked some unrelated changes here.

Reply:
We just merged the latest master branch to our code. The file `.github/checkgroup.yml` is the same as the parallel file in the latest master branch. I don't know why this problem is occured.

Reply:
The problem is solved when I rebased my branch to the master branch.